src/ai/assistant.ts

```diff
@@ -435,4 +436,13 @@ export class Assistants extends FactoryBase {
       return 0;
     });
   }
+
+  private countThreads(items: any) {
+    if (typeof items === 'boolean' || !items) {
+      return 0;
+    }
+    return _.sumBy(_.values(items), (item) => {
```

Comment:
minor) is lodash util dependency required? for example, can reduce() and Object.keys(), and Object.values() be used directly? ref: https://github.com/ainblockchain/ain-blockchain/blob/master/db/functions.js#L900

Reply:
done!